integration-tests/project-folder/sources/meta-error/recipes-error/error/unparsed-line.bb

```diff
@@ -0,0 +1,6 @@
+# NOTE:
```

Comment:
Great! You'll also have some errors that only happen if they are written in the layer.conf or a distro.conf. They'll produce different error messages to! You'll have to use separate layers instead of BBMASK if you want to test them as well.

Reply:
It's also nice to have the example errors as a comment.

Reply:
Could you provide some examples of the errors in `layer.conf` or other conf files?

Reply:
Basically the same syntax errors you have in .bb file. If you put them in layer.conf or a distro.conf, you get different errors to parse !

Reply:
I lost my examples unfortunately. Could you reproduce every kind of error described in the problems matcher? I can help reproducing the other ones if not the case by injecting syntax errors deep in the .conf or library. Ex:
`poky/meta/lib/oe/__init__.py`:
```
a=2+'2'
```
```
ERROR: Unable to parse /home/deribaucourt/Workspace/yocto-vscode/yocto/yocto-build/sources/poky/meta/lib/oe/__init__.py
```

Reply:
I think I am missing the errors for `bitbake-variable` and `bitbake-task-error`, what could cause those errors?

Reply:
bitbake-task-error `yocto-build/sources/poky/meta/recipes-core/base-files/base-files_3.0.14.bb`:
```bitbake
do_install:append () {
    bbfatal should crash
}
```
But to reproduce this one, you have to use `.vscode/tasks.json`. It's only for build tasks:
```json
{
    "tasks": [
        {
            "type":"bitbake",
            "recipes": ["base-files"],
        }
    ]
}
```
Resulted in this:
`ERROR: Task (/home/deribaucourt/Workspace/yocto-vscode/yocto/yocto-build/sources/poky/meta/recipes-core/base-files/base-files_3.0.14.bb:do_install) failed with exit code '1'`

Reply:
God it was hard to remember `bitbake-variable` but I got it!
```
BB_ENV_EXTRAWHITE = "deprecated var"
```
I get `Parsing recipes...ERROR: Variable BB_ENV_EXTRAWHITE has been renamed to BB_ENV_PASSTHROUGH_ADDITIONS (file: /home/deribaucourt/Workspace/yocto-vscode/yocto/yocto-build/sources/poky/meta/recipes-core/base-files/base-files_3.0.14.bb line: 194)`

Reply:
From data_smart.py _print_rename_error

Reply:
Thanks. Added